extensions/amp-analytics/0.1/variables.js

```diff
@@ -220,7 +220,7 @@ export class VariableService {
         // Expand string values further.
         p = this.expandTemplate(raw,
             new ExpansionOptions(options.vars, options.iterations - 1,
-                options.noEncode));
+                true /* noEncode */));
```

Comment:
Perfect, now about that test. Is that return value `123${4}` encoded at some point? If not, that's the XSS I'm talking about.